python/README.md

```diff
@@ -57,7 +57,7 @@ Disable:
 $ python3 recognize.py -r 16000 -c 1 -e MPEG_AUDIO --disable_profanity_filter ../audio/sample_5.mp3
 ```
 
-Also there is a REST-like API for non-streaming speech recognition:
+Also, there is a REST-like API for non-streaming speech recognition (if you are planning to use REST in production, which is not recommended, by the way, make sure to use libraries which support HTTP2, like httpx):
```

Comment:
Is this relevant to the ListVoices method?

Reply:
I think in general it would be better to use http2 everywhere